react/src/components/atoms/buttons/ButtonAlert/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import is from 'is';
+import PropTypes from 'prop-types';
+import './style.css';
+
+const ButtonAlert = ({
+  text, onClick, hideText, showText, classes
+}) => {
+  const buttonProps = {
+    className: buttonClasses
```

Comment:
I think this should be this instead:
```suggestion
    className: 'ma__button-alert'
```
This allows for buttonClasses to not need a different variable later.